app/controllers/submissions_controller.rb

```diff
@@ -578,6 +581,40 @@ def download_groupings_files
   end
 
   ##
+  # Check the status of collection for all groupings
+  ##
+  def check_collect_status
+
+    assignment = Assignment.find(params[:assignment_id])
+    grouping_ids = params[:groupings]
+
+    ## if there is no grouping, render a message
+    if grouping_ids.blank?
+      render text: t('student.submission.no_groupings_available')
+      return
+    end
+
+    groupings = Grouping.where(id: grouping_ids)
+                    .includes(:group,
+                              current_submission_used: {
+                                  submission_files: {
+                                      submission: { grouping: :group }
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.